Makefile

```diff
@@ -34,13 +40,14 @@ download-ext:
 clean-ext:
 	$(MAKE) -C src_ext distclean
 
-clean: fastclean
```

Comment:
Actually, I think I originally made this change before you committed e141c2e to fix #2867.

It's up to you if you'd like this changed to `rm -rf _obuild/` instead but for me, the underlying is fine (because it works!)